Modules/IO/MeshBase/include/itkMeshIOBase.h

```diff
@@ -382,8 +382,8 @@ class ITKIOMeshBase_EXPORT MeshIOBase:public LightProcessObject
   itkGetConstMacro(NumberOfPointPixelComponents, unsigned int);
   itkSetMacro(NumberOfCellPixelComponents, unsigned int);
   itkGetConstMacro(NumberOfCellPixelComponents, unsigned int);
-  itkSetMacro(PointDimension, unsigned int);
-  itkGetConstMacro(PointDimension, unsigned int);
+  itkSetMacro(PointDimension, SizeValueType);
+  itkGetConstMacro(PointDimension, SizeValueType);
```

Comment:
For this particular one, I'm not sure this should be changed. For things link number of image dimensions and number of components we are still using a hard coded "unsigned int" type. This appear to be the same kind of type from reading the documentation: https://itk.org/Doxygen/html/classitk_1_1MeshIOBase.html#aa8cbbad10bc2781fd77cdc118ae79da8

Reply:
@blowekamp Thanks for your reply. But `MeshIOBase::m_PointDimension` itself is defined as `SizeValueType`! At https://github.com/InsightSoftwareConsortium/ITK/blob/v5.0.0/Modules/IO/MeshBase/include/itkMeshIOBase.h#L737

      SizeValueType m_PointDimension{3};

Reply:
Then perhaps the correct fix is to change the data type of the IVAR? It may be less disruptive to the API and class users.

Reply:
@blowekamp The last force-push I did yesterday includes your suggestion: it fixes a "loss of data warning" by declaring `m_PointDimension` as `unsigned int`, instead of `SizeValueType`. OK?

Reply:
Great! I think we are just waiting on getting the windows CI back now...